llama_stack/providers/impls/meta_reference/scoring/scorer/inclusion_scorer.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from llama_stack.providers.impls.meta_reference.scoring.scorer.base_scorer import (
+    BaseScorer,
+)
+from llama_stack.apis.scoring_functions import *  # noqa: F401, F403
+from llama_stack.apis.scoring import *  # noqa: F401, F403
+from llama_stack.apis.common.type_system import *  # noqa: F403
+from llama_stack.providers.impls.meta_reference.scoring.scorer.common import (
+    aggregate_accuracy,
+)
+
+
+class InclusionScorer(BaseScorer):
```

Comment:
is there an accepted name for this evaluator? "Inclusion" feels a bit odd even though I get it.